docs/release_notes.rst

```diff
@@ -35,6 +35,8 @@ Features
   (`#1321 <https://github.com/fedora-infra/bodhi/issues/1321>`_).
 * The bodhi CLI now supports editing an update using the update id.
   (`#937 <https://github.com/fedora-infra/bodhi/issues/937>`_)
+* The bodhi CLI now supports editing an override using the nvr.
```

Comment:
I think we can leave off the "using the nvr" because it makes it sound like it was possible to edit overrides some other way before (and thus minimizes how great this change is!)